src-opam/dockerfile_distro.ml

```diff
@@ -179,7 +182,7 @@ let win10_docker_status (base : win10_docker_base_image) v : status =
   | `NanoServer, `V1607 -> `Deprecated
   | _ -> `Not_available
 
-let win10_latest_image = `V20H2
+let win10_latest_image = `Ltsc2022
```

Comment:
Conversely this one is fine, if now slightly strangely named!

Reply:
Leaving it as-is, then!